doc/man1/openssl-verification-options.pod

```diff
@@ -682,6 +705,8 @@ L<openssl-cms(1)>
 
 =head1 HISTORY
 
+This B<-trusted_first> option is always enabled since since OpenSSL 1.1.0.
```

Comment:
This this instead:

_Since OpenSSL 1.1.0, the B<-trusted_first> option is always enabled._